filters-share/specific_ELEMHIDE.txt

```diff
@@ -1,3 +1,4 @@
+www.instiz.net##.hot_under
```

Comment:
Where?

<details>
<summary>Screenshot</summary>

![Scree222nshot](https://user-images.githubusercontent.com/98787049/163720153-be312e0b-a561-4f99-884a-e04a9082cd54.png)

</details>

Reply:
1. 
![image](https://user-images.githubusercontent.com/34261355/163720261-3383c024-8b4a-4512-b5ab-03c43149a344.png)

2.
![image](https://user-images.githubusercontent.com/34261355/163720266-00ddbbc5-7aec-4ade-911e-623602a0b58e.png)

Reply:
I cloud not find.
Please share the elements in `.hot_under`.

Reply:
![image](https://user-images.githubusercontent.com/34261355/163720446-dc0f1c76-c240-4963-8e0d-27b465cef616.png)
![image](https://user-images.githubusercontent.com/34261355/163720449-47457619-db16-485c-9f77-9140c86f78dd.png)